src/evm-instructions.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Hera VM: eWASM virtual machine conforming to the Ethereum VM C API
+ *
+ * Copyright (c) 2016 Alex Beregszaszi
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+#pragma once
+
+namespace HeraVM {
+
+const std::map<uint8_t, std::string> evmInstructionNames;
```

Comment:
This map will be dynamically initialized also when never used.

Reply:
Is that a lot memory?

Reply:
Any suggestions?

Reply:
Flat array.
But the map can stay for now. I will think about having this in EVMC because every VM implementations needs it.

Reply:
See https://github.com/ethereum/cpp-ethereum/issues/4072.

Reply:
Is that an array of `char*` and for the missing ones filled with `NULL` or `"UNKOWN"`? If it goes into EVMC it has to be C.